extensions/jms/src/main/java/org/mule/extensions/jms/internal/message/JmsMessageUtils.java

```diff
@@ -67,6 +70,11 @@ public static Message toMessage(Object object, Session session) throws JMSExcept
     } else if (object instanceof InputStream) {
       return inputStreamToMessage((InputStream) object, session);
 
+      // TODO MULE-12270 : SDK Should provide a generic way to handle CursorStreamProviders
```

Comment:
won't fix. This needs to be resolved at the runtime level. Extensions should absolutelly not need to know about the providers.